morphic.js

```diff
@@ -5056,28 +5072,47 @@ CursorMorph.prototype.processKeyPress = function (event) {
 
 CursorMorph.prototype.processKeyDown = function (event) {
     // this.inspectKeyEvent(event);
-    var shift = event.shiftKey;
+    var shift = event.shiftKey,
+        wordNavigation = event.ctrlKey || event.altKey,
+        selecting = this.target.selection().length > 0;
```

Comment:
little quib: How about getting right of the ```> 0``` part, because if the ```length``` is zero, ```selecting``` will be falsey already, right? :-)